Makefile

```diff
@@ -121,9 +121,11 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	$(KUSTOMIZE) build config/operand_rbac | kubectl apply -f -
 
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
+	$(KUSTOMIZE) build config/operand_rbac | kubectl delete -f -
```

Comment:
Sorry, I think told you the wrong thing on the previous PR.  We should delete the operand first so that the operator can clean things up, and then we should delete the operator namespace.

Reply:
Sure, sounds more logical to me as well